web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,35 @@
+# Downloading Diagnostics
+
+Users can download a diagnostic report of their system and send it to Red Hat from the console.
+
+
+## Help menu
+![dropdown](img/dropdown.png)
+* The "Download Diagnostics" option is available from the help menu.
+
+## Modal
+![modal](img/modal.png)
+* A modal will open with a description of how the diagnostic report will be used.
+* Users can indicate what they would like to include in the diagnostic report and whether or not they have an open support case already.
+  * If users have a case number they can enter it into the "Case number" field.
+  * If users do not have an open case they may choose to open one. If they choose this option, a link to the case will be included in the success screen.
+  * Users may also choose to just download the diagnostic report to their machine and send it to Red Hat at a later time.
+
+## In progress
+![generating](img/generating.png)
```

Comment:
Would this modal stay the same size and just change the content as well as the cancel button changing to close? Or is it a separate popup?

Reply:
Cancel vs close here may depend on whether the process can be stopped once initiated.

Reply:
Yeah it would depend on whether it can be stopped. I would leave the button to read "Close", and if the download can be stopped I can add a separate "Cancel download" button. Since this can take a while I think it's important to let users close this modal and do other things in the meantime, hence the toast notification if the modal is not open.

I don't think a double-modal situation would be great, I imagined this modal would change size as content changes. I think it's a standard interaction, but let me know if you disagree.